packages/calcite-design-tokens/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [1.0.0](2023-05-11)
+
+### Features
+
+- uses @token-studio tokens format
+- add css platform
+- add scss platform
```

Comment:
Can you make all of these items sentence case for consistency? We have a `release-please` plugin that does that for us when creating the changelogs.

https://github.com/Esri/calcite-design-system/blob/93d67a3a9ac0bf376661722538959d2aff0383bd/release-please-config.json#L28-L30